entropylab/results/dashboard/auto_plot.py

```diff
@@ -14,6 +15,8 @@ def auto_plot(experiment_id: int, data):
         plot = _auto_plot_from_list(data)
     elif isinstance(data, np.ndarray):
         plot = _auto_plot_from_ndarray(data)
+    elif isinstance(data,int) or isinstance(data,float):
```

Comment:
Nitpicking: Missing white space after commas (,). :)

If you'd like I can show you how to have `black` fix this for you automatically in PyCharm.

Reply:
yeah, if we go over it together we can go over both black and unit-test
thanks!